pkg/proxy/endpoints_test.go

```diff
@@ -1709,15 +1690,17 @@ func TestCheckoutChanges(t *testing.T) {
 				t.Fatalf("Expected %d changes, got %d", len(tc.expectedChanges), len(changes))
 			}
 
-			for i, change := range changes {
-				expectedChange := tc.expectedChanges[i]
+			for _, change := range changes {
+				// The test only supports 0 or 1 changes, so if we're here,
```

Comment:
I didn't check in detail , but is this because the test can not support more than 1 change or existing testcases only have 0 or 1?

Reply:
the latter